FlexMeasures/flexmeasures
# Asset audit log

## Description

Start adding audit log records for asset related actions.

## Look & Feel

New router assets/<id>/auditlog shows account audit log records

## How to test

Create an asset, edit it, add / remove sensors, go to auditlog and these actions should be displayed there

## Further Improvements

-

## Related Items

-

---
